pkg/injector/injector.go

```diff
@@ -117,8 +121,25 @@ func NewInjector(authUIDs []string, config Config, daprClient scheme.Interface,
 		authUIDs:   authUIDs,
 	}
 
+	matcher, err := createNamespaceNameMatcher(strings.TrimSpace(config.AllowedServiceAccountsPrefixNames))
+	if err != nil {
+		return nil, err
+	}
+	i.namespaceNameMatcher = matcher
+
 	mux.HandleFunc("/mutate", i.handleRequest)
-	return i
+	return i, nil
+}
+
+func createNamespaceNameMatcher(allowedPrefix string) (matcher *namespacednamematcher.EqualPrefixNameNamespaceMatcher, err error) {
+	if prefixNames := allowedPrefix; prefixNames != "" {
```

Comment:
Could this be just:

```suggestion
	if allowedPrefix != "" {
```

Reply:
sure, it must have done something before a refactor and was left dangling. thanks

Reply:
done